src/ExpressionEvaluator/Core/Source/ExpressionCompiler/MetadataUtilities.cs

```diff
@@ -178,9 +175,24 @@ internal static string GetFileNameWithoutExtension(string fileName)
             return fileName;
         }
 
+        private static byte[] GetWindowsProxyBytes()
+        {
+            var assembly = typeof(ExpressionCompiler).GetTypeInfo().Assembly;
+            var resourceName = assembly.GetManifestResourceNames().Single();
+            using (var stream = assembly.GetManifestResourceStream(resourceName))
```

Comment:
Frankly, I did it this way since I didn't know the name.  I've extracted it from the debugger and hard-coded it.